apps/codeforafrica/src/components/FooterMenu/index.js

```diff
@@ -0,0 +1,52 @@
+import { Link } from "@commons-ui/next";
+import PropTypes from "prop-types";
+import React from "react";
+
+import NavList from "@/codeforafrica/components/NavList";
+import NavListItem from "@/codeforafrica/components/NavListItem";
+
+function FooterMenu({ menu }) {
+  if (!menu?.length) {
+    return null;
+  }
+
+  return (
+    <NavList direction="column">
```

Comment:
```suggestion
    <Box
      component="nav"
      sx={{
        display: "flex",
        justifyContent: {{ xs: "center", md: "flex-end" }}
    >
      <NavList direction="column">
```

`Box` is just an example. You can use any other component to align them (but remember to set `component="nav"` for accessibility, etc.